runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -298,6 +298,13 @@ J9::CodeGenerator::fastpathAcmpHelper(TR::Node *node, TR::TreeTop *tt, const boo
    prevBlock->append(TR::TreeTop::create(comp, ifacmpeqNode));
    }
 
+bool J9::CodeGenerator::supportsMergingGuards()
```

Comment:
Why do we have this override the implementation looks the same...

Reply:
As per the discussion in https://github.com/eclipse/omr/pull/4456#discussion_r450334728, we are removing `allowGuardMerging()` from OMR and into OpenJ9 since it always returns false in OMR and has no other use there. Given that, the OMR implementation in the above mentioned PR has to be changed to no longer use `allowGuardMerging()`. So the original OMR implementation is being brought into OpenJ9, and the OMR implementation is being changed in https://github.com/eclipse/omr/pull/4456.